pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -296,7 +303,8 @@ func (c *EtcdCertSignerController) createSecretForNode(node *corev1.Node) error
 	if err != nil {
 		return err
 	}
-	peerHostNames := append([]string{"localhost", etcdDiscoveryDomain}, nodeInternalIPs...)
```

Comment:
what happened to etcdDiscoveryDomain?

Reply:
it was invalid and a typo

Reply:
what we wanted was a wildcard. In 4.4+ we no longer need DNS in SAN as etcd only uses IP. So new defaults are localhost to allow quorum-guard to function and  nodeInternalIPs

Reply:
we then append compatability records for 4.3